Project-Aurora/Project-Aurora/Profiles/Witcher3/GameEvent_Witcher3.cs

```diff
@@ -66,79 +63,64 @@ public override void ResetGameState()
             _game_state = new GameState_Witcher3();
         }
 
-        public new bool IsEnabled
-        {
-            get { return this.Application.Settings.IsEnabled && isInitialized; }
-        }
-
         public override void UpdateLights(EffectFrame frame)
         {
             Queue<EffectLayer> layers = new Queue<EffectLayer>();
 
             if (File.Exists(dataPath))
             {
-                if(configContent != null)
```

Comment:
How often does the config change? If it is not changing as often as each frame ticks, this code here will be running unnessecarily.

You could define a boolean field for whether the game state is 'dirty'. This could be set to true every time the config file is re-read and then only run this part of the code if `configContent != null && isStateDirty`?

Obviously if the config is being written very regularlly, this wouln't achieve anything. And again, if performance is okay at the moment this is maybe a non-issue.

Reply:
The mod writes to te config every 30 player ticks. I'm not sure what exactly this means but seems to be less often than an aurora frame tick. I implemented this with the latest commit.